ingenialink/eoe/network.py

```diff
@@ -56,14 +56,15 @@ def connect_to_slave(self, slave_id, ip_address, dictionary=None,
 
         """
         self._configure_slave(slave_id, ip_address)
+        if not self._eoe_service_started:
+            self._start_eoe_service()
```

Comment:
What happens if you run a script twice and the service was started in the first run?

Reply:
Nothing, you can re-run the script without a problem.